{{cookiecutter.project_slug}}/.osparc/bin/ooil.bash

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+
+set -o errexit
+set -o nounset
+set -o pipefail
+IFS=$'\n\t'
+
+IMAGE_NAME="itisfoundation/service-integration:${OOIL_IMAGE_TAG:-master-github-latest}"
+WORKDIR="$(pwd)"
+
+run_with_color() {
+  docker run \
+    -it \
```

Comment:
1. you should use long option names
2. minor: you could also pass this with a parameter, that is the only difference... and also I am not sure I see the link with the colors ;)

Reply:
yes. I did this separation as a reminder and then ... i forgot :-(
The problem is that with colors, when you  ``echo ooil --version >VERSION`` it also stores the codes of the colors in ``VERSION`` :-D ...